deployments/charts/kuma/templates/cp-deployment.yaml

```diff
@@ -97,6 +97,9 @@ spec:
       {{- with .Values.controlPlane.topologySpreadConstraints }}
       topologySpreadConstraints: {{ tpl (toYaml . | nindent 8) $ }}
       {{- end }}
+      {{- with .Values.controlPlane.priorityClassName }}
```

Comment:
Could you also add this for Ingress and Egress?

Reply:
Done!